### PR TITLE
Eliask.get post put delete

### PR DESCRIPTION
**Pull request by ekraihan**
It would be great if you could check the name and casing of the parameters and return values of my functions to make sure they all line up with what ya'll are expecting

### CODE REVIEW

File: retina-api/retina-api/Controllers/SearchController.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            myConnection.Open();
+
+            SqlCommand cmd;
+            if (searchType == "toolboxsearch")
+            {
+                cmd = new SqlCommand("?????", myConnection);
+            }
+            else
+            {
+                cmd = new SqlCommand("??????", myConnection);
```
Comment (Jbcampbe) [inline retina-api/retina-api/Controllers/SearchController.cs]: so i take it these aren't wired up quite yet

File: retina-api/retina-api/Controllers/TokenController.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+namespace retina_api.Controllers
+{
+    public class TokenController : ApiController
+    {
+
+        private string generateToken()
+        {
+            
+            return "blah";
```
Comment (Jbcampbe) [inline retina-api/retina-api/Controllers/TokenController.cs]: I have no idea what kevin would do but I have a feeling I wouldn't want to see the look on his face when he sees this 😝

File: retina-api/retina-api/Controllers/SearchController.cs (hunk 1 above)
Comment (JerebChase) [inline retina-api/retina-api/Controllers/SearchController.cs]: Yeah, I'm working on those guys currently...

File: retina-api/retina-api/Controllers/TokenController.cs (hunk 2 above)
Comment (ekraihan) [inline retina-api/retina-api/Controllers/TokenController.cs]: oh, yea :)

File: retina-api/retina-api/Controllers/TokenController.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+            SqlCommand cmd = new SqlCommand("authenticate_user", myConnection);
+            cmd.CommandType = CommandType.StoredProcedure;
+
+            cmd.Parameters.AddWithValue("@UserName", requestBody.identification);
+            cmd.Parameters.AddWithValue("@Password", requestBody.password);
+
+            SqlDataReader myReader = cmd.ExecuteReader();
+
+            while (myReader.Read())
```
Comment (Jbcampbe) [inline retina-api/retina-api/Controllers/TokenController.cs]: is it possible for this to be an `if`?

Comment (ekraihan) [inline retina-api/retina-api/Controllers/TokenController.cs]: I believe so, but i wanted to test out how ```.Read()``` works before changing it to that. But i can actually test that now

**Pull request merged**